Tests/ClientTests/TabManagement/TabManagerTests.swift

```diff
@@ -92,6 +92,38 @@ class TabManagerTests: XCTestCase {
         XCTAssertEqual(subject.tabs.count, 5)
     }
 
+    // MARK: - Save preview screenshot
+
+    func testSaveScreenshotWithNoImage() async throws {
+        addTabs(count: 5)
+        if let tab = subject.tabs.first {
```

Comment:
We should have an `else` that calls `XCTFail` in the test to ensure we go into the right path (for the 3 test cases)